src/objects.js

```diff
@@ -1,41 +1,67 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    'name': name,
+    'age': age 
```

Comment:
A minor point, but you can use the assignment shorthand for this:
```suggestion
    name,
    age
```